protocol/x/rewards/keeper/keeper_test.go

```diff
@@ -511,36 +534,40 @@ func TestProcessRewardsForBlock(t *testing.T) {
 					Weight:  dtypes.NewInt(30_000_000), // $30 weight of fee
 				},
 			},
-			tokenPrice:             tokenPrice2Usdc,
-			treasuryAccountBalance: sdkmath.NewInt(10_000_000), // 10 full coins
-			feeMultiplierPpm:       1_000_000,                  // 100%
+			tokenPrice: tokenPrice2Usdc,
+			treasuryAccountBalance: sdkmath.NewIntFromBigInt(
+				big_testutil.Int64MulPow10(10, 18),
+			), // 10 full coins
+			feeMultiplierPpm: 1_000_000, // 100%
 			expectedBalances: []banktypes.Balance{
 				{
 					Address: TestAddress1,
 					Coins: []sdk.Coin{{
 						Denom:  TestRewardTokenDenom,
-						Amount: sdkmath.NewInt(1_666_666), // 1/6 of 10 = 1.666666 full coins
+						Amount: sdkmath.NewInt(1666666666666666666), // 1/6 of 10 = 1.666666 full coins
```

Comment:
nit: could use underscores here for legibility
```suggestion
						Amount: sdkmath.NewInt(1_666_666_666_666_666_666), // 1/6 of 10 = 1.666666 full coins
```